lib/publisher/apppublisher.go

```diff
@@ -0,0 +1,40 @@
+package publisher
+
+type RemoteAppType string
+
+const (
+	Webapp     RemoteAppType = "REMOTE_WEB_APP"
+	Server     RemoteAppType = "REMOTE_SERVER"
+	Database   RemoteAppType = "REMOTE_DATABASE"
+	Desktop    RemoteAppType = "REMOTE_DESKTOP"
+	Kubernetes RemoteAppType = "REMOTE_KUBERNETES"
+
+	//default sqs queue name
+	defaultSQSQueueName = "remote-access-resource-change-notification-queue"
+)
+
+type AppChangeEvent struct {
+	AppType RemoteAppType `json:"appType,omitempty"`
+	Tenant  string        `json:"tenant,omitempty"`
```

Comment:
do you want to name it as TenantUrl so it is clear that it is the tenant url we are talking about.